clippy_lints/src/large_stack_arrays.rs

```diff
@@ -1,3 +1,5 @@
+use std::num::Saturating;
+
```

Comment:
Out of curiosity (it's too late now anyway), is the saturating behavior actually necessary here? I don't see how it could underflow - that sounds like it would somehow require `check_item_post` to run before `check_item` or some other weird out of order visiting (which would break this anyway)?

Reply:
I was just being really defensive, in case of weird shenanigans.